### PR TITLE
Voting map proof of concept

### PR DESCRIPTION
**Pull request by dbpokorny**
Here is a proof of concept of the proposed alternative to the algebraic decision process for voting

### CODE REVIEW

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing :+1:`:+1:`, or :-1:`:-1:`.
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both :+1:up- and :-1:down-votes are disregarded.
Comments containing monkeys are disregarded. :monkey: :monkey_face: :see_no_evil: :hear_no_evil: :speak_no_evil:
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **8** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:*
 ca7fcd6ddb8d6c77eb44506f884359ee9a63025e

**Comment by samis** [comment]
Interesting idea. What if the vote count is larger than 9 though?

File: lib/voting/voting.js [hunk 1]
```diff
@@ -34,9 +34,40 @@
...
 
+        // A voting map must satisfy the following constraints:
+        //
+        // (0) a-b means a upvotes and b downvotes
+        // (1) a policy mark is one of . L W
+        // (5) a . mark means it is not possible to come to a decision yet
+        // (6) marks L and W signify loss and win
+
+        var VOTE_SIZE = 9;
+        var VOTE_MAP = [
```
Comment (RyanCopley) [inline lib/voting/voting.js]: I don't see any advantage over the old system... :-1:

**Comment by RyanCopley** [comment]
@samis https://github.com/botwillacceptanything/botwillacceptanything/pull/405/files#diff-3879e5fcefae9b1f342ef7b2446a11b8R61 is what would stop it... but it's mucky code. :-1:

**Comment by dbpokorny** [comment]
RyanCopley: please see this thread: https://github.com/botwillacceptanything/botwillacceptanything/pull/397

**Comment by SpenserJ** [comment]
:-1: Not a fan of building a map of vote results, when a simple algorithm is easier to follow and maintain.

**Comment by korczis** [comment]
:-1:

**Comment by thenightwassave** [comment]
:-1: Agree with spencer

**Comment by OLef1air** [comment]
:-1:

**Pull request closed without merging**